pepper/test/apps/isBoolVerification_test.cpp

```diff
@@ -0,0 +1,32 @@
+#include <cassert>
+#include <gtest/gtest.h>
+#include "declarations.h"
+#include <apps/hash_transform.h>
+
+#include <apps/dex_common.h>
+#include "util.h"
+
+struct TestStructArray { field digest[256]; };
+
+TEST(isBoolVerification, isBoolVerificationWithBools) { 
+    uint32_t length = 2;
+    struct TestStructArray bits[1] = { 0 };
```

Comment:
Why do you need a struct here? Can't you just do:
```
field  digest[256] = { 0 }
EXPECT_NO_THROW(isBoolVerification(digest, length));
```

Reply:
yeah... changed it.